src/app/components/cohort-grid/cohort-grid.less

```diff
@@ -1,3 +1,7 @@
 .ts-export-btn {
   float: right;
 }
+.ts-cohort-grid {
+  width: 600px;
```

Comment:
just wondering is it possible to make the width is adjusted when a node is added? e.g. .. add first node width = 200px, add 2nd node width = 400px;